src/CompatHelper.jl

```diff
@@ -9,6 +9,7 @@ using Pkg: Pkg
 using TimeZones: TimeZones
 using TOML: TOML
 using UUIDs: UUIDs, UUID
+using RegistryInstances
```

Comment:
```suggestion
using RegistryInstances: RegistryInstances
```

Reply:
I'm not familiar with this pattern. What does it do?

Reply:
It's equivalent to `import Foo`.

The idea is that we do not want to bring all of the exported names into scope.

Reply:
Makes sense. I thought you had to use import for that and indeed it's misleading if you don't plan to extend anything. I'll change this and also the tests tomorrow.